### PR TITLE
Model refinement doc

### PR DESCRIPTION
**Pull request by smartin015**
This guide is mostly speculative - don't merge this PR. #589

I haven't yet done the legwork to train any models yet. I wanted you to see it early to let me know if there's anything you'd like to change before I actually start writing scripts and running code. 

A couple specific things I'd like feedback on:

* If you'd prefer I embed example code for running commands (vs checking in a script) let me know and I'll adjust.
* I cautioned folks from contributing fine-tuned models for their particular environment. Would you rather this, or that they open a pull request so you can see what they did?
* Do you have any guidance on a reasonable learning rate for fine tuning? 0.0005 was a stab in the dark.

### CODE REVIEW

**Comment by kennethjiang** [comment]
Love the direction you are heading in. Let me know when it's ready for review.

**Comment by kennethjiang** [comment]
@smartin015 I feel this PR is quite good. Do you want me to merge it?

**Comment by smartin015** [comment]
Hey Kenneth - let me quickly clean up some of the todo sections today and I'll have it ready for merge, thanks :)

**Comment by smartin015** [comment]
Ok, I've stripped out the todos. IMO it's missing some handholding in preparing the training data, but it should be at least a good overall reference that folks can use to get started and hopefully improve on.

**Comment by kennethjiang** [comment]
Thank you so much @smartin015
